src/services/api/fantlab/book.ts

```diff
@@ -77,3 +79,20 @@ function children(w) {
     year: c.work_year,
   }));
 }
+
+function editionIds(w) {
+  const RUSSIAN_EDITION = 10
+  return _.get(w, `editions_blocks.${RUSSIAN_EDITION}.list`, []).map(el => el.edition_id)
```

Comment:
Is it really works? :) I usually use that syntax:
```JS
_.get(w, ['editions_blocks', RUSSIAN_EDITION, 'list'], [])
```

Reply:
Well, yeah, as you can see :D I usually use it this way 🤷‍♀